grunt_tasks/build.js

```diff
@@ -80,6 +80,14 @@ module.exports = function (grunt) {
                     src: ['img/**', 'jsoneditor.min.css'],
                     dest: 'clients/web/static/built/jsoneditor'
                 }]
+            },
+            'eonasdan-bootstrap-datetimepicker': {
+                files: [{
+                    expand: true,
+                    cwd: 'node_modules/eonasdan-bootstrap-datetimepicker/build',
+                    src: 'css/bootstrap-datetimepicker.min.css',
+                    dest: 'clients/web/static/built/eonasdan-bootstrap-datetimepicker'
+                }]
```

Comment:
Sorry for not noticing this earlier... rather than copying this, we can actually pass `node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.css` into our stylus inputs for `app.min.css`. The nice thing about doing it that way is that is it prevents our webroot and downstream webroots (such as COVALIC's) from needing to serve an extra file in their mako templates.